ui/redux/selectors/collections.js

```diff
@@ -232,6 +239,11 @@ export const makeSelectNameForCollectionId = (id: string) =>
     return (collection && collection.name) || '';
   });
 
+export const makeSelectUpdatedAtForCollectionId = (id: string) =>
+  createSelector(makeSelectCollectionForId(id), (collection) => {
+    return (collection && collection.updatedAt) || '';
+  });
+
```

Comment:
No more `makeSelect*` pattern for new selectors -- it doesn't really cache -- it creates a new cache each time and never re-uses it. so it somewhat a memory leak.  If you do an infinite for-loop that calls a `makeSelect*`, you will see the memory usage goes GB until garbage collection kicks in during idle.

Do `select*(state, id)` instead, and can make it non-caching for simplicity, until caching is really needed.  Inside it, you can call existing `makeSelect` selectors if you do not wish to clean those up yet.